samples/extensions/README.adoc

```diff
@@ -262,8 +262,14 @@ Demonstrate how to use the blending related functions available in the VK_EXT_ex
 
 Demonstrate methods for dynamically customizing the appearance of the rendered lines.
 
+=== xref:./{extension_samplespath}shader_debugprintf/README.adoc[Shader Debug Printf]
+
+*Extension*: https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_shader_non_semantic_info.html[`VK_KHR_shader_non_semantic_info`]
+
+Demonstrates how to use https://en.wikipedia.org/wiki/Printf[Printf] statements in a shader to output per-invocation values. This can help find issues with shaders in combination with graphics debugging tools.
+
```

Comment:
Isn't this text part of a different PR?

Reply:
Yes, but this is weird. As you see I didn't change any letter even line is the same. After that line I added new line with dynamic primitive clipping. To be more elegant I can fix that to doesn't have that line in git diff.